src/Sil/Component/Product/Model/Product.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2015-2018 Libre Informatique
+ *
+ * This file is licenced under the GNU LGPL v3.
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Sil\Component\Product\Model;
+
+use Blast\Component\Resource\Model\ResourceInterface;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\Common\Collections\ArrayCollection;
+
+class Product implements ProductInterface, ResourceInterface
```

Comment:
Should ResourceInterface be extended directly by ProductInterface ?

Reply:
`Product` should not be used as it is, you're right.